templates/blog/post_detail.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
+{% load crispy_forms_tags %}
+
+{% block content %}
+<h1>{{ post.title }}</h1>
+<p>{{ post.content }}</p>
+<h3>{{ post.commentary_count }} comment{{ post.commentary_count|pluralize }}:</h3>
+<ul>
+    {% for comment in post.commentary.all %}
```

Comment:
There is a potential issue here: `post.commentary.all` assumes that `commentary` is a related name for comments in the `post` model. Ensure that the related name is correctly set in the model definition.